### PR TITLE
summit training day announcement

### PR DESCRIPTION
**Pull request by bbrcknl**
Preliminary draft. Needs input from @Ivan-Velickovic and Nick Spinale. Logos also needed. Accompanying news article needed.

### CODE REVIEW

**Review by Ivan-Velickovic — APPROVED** [review]
Looks good to me. We are planning to have a LionsOS logo but not sure when we'll have it ready so I'll make a PR adding it to this page later on.

**Comment by Ivan-Velickovic** [comment]
Oh, we should sort out the time as well.

Something like 10AM - 12:30PM would be good for LionsOS. Then Rust in the afternoon at 1:30PM or so?

**Comment by bbrcknl** [comment]
> Looks good to me. We are planning to have a LionsOS logo but not sure when we'll have it ready so I'll make a PR adding it to this page later on.

That would be great!

**Comment by june-andronick** [comment]
Please add a link back to the descriptions from the items in the program's page. Thanks :)

**Comment by nspin** [comment]
I suggest that we change the name of the Rust training session to "Using Rust in seL4 Userspace".

Also, how about this for the text in that section:

Rust has been an officially supported language for use in seL4 userspace since last year's summit. In this workshop, participants will be led through a tutorial that demonstrates how to leverage Rust and its ecosystem to rapidly build memory-safe components for seL4-based systems. This tutorial will cover a range of scenarios, including <a href="https://docs.sel4.systems/projects/microkit/">Microkit</a> protection domains as well as building on seL4 directly.

File: Foundation/Summit/2024/index.html [hunk 1]
```diff
@@ -20,14 +20,13 @@ <h4>
...
+    <li><a href="#summit-registration">Registration</a></li>
     <li><a href="#summit-location">Location & Venue</a></li>
     <li><a href="#summit-cfp">Submissions and Call for Presentations</a></li>
+    <li><a href="#summit-program">Program & Abstracts</a></li>
     <li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
     <!-- li><a href="#summit-panellists">Panellists</a></li>-->
-    <li><a class="highlight" href="#summit-program">Latest news: Program & Abstracts</a></li>
-    <!--li><a href="#summit-keynote-speakers">Keynote Speakers</a></li>
-    <li><a href="#summit-panellists">Panellists</a></li-->
+    <li><a class="highlight" href="#summit-training-day">Latest news: Training day @UNSW</a></li>
```
Comment (Ivan-Velickovic) [inline Foundation/Summit/2024/index.html:29]:
```suggestion
    <li><a class="highlight" href="#summit-training-day">Latest news: Training day @ UNSW</a></li>
```

should this have a space between the "@" and "UNSW"?

File: Foundation/Summit/2024/index.html [hunk 2]
```diff
@@ -175,6 +173,77 @@ <h3 id="summit-keynote-speakers">
...
+  </p>
+  <h4 style="padding-top:10px">Registration for training day</h4>
+  <p>
+    <a href="https://form.jotform.com/242312058251850">Register for the LionsOS, Rust or both tutorials</a>
+  </p>
+  <p>
+      Training day @UNSW is free, but please register by <span class="highlight"> 15 September 2024</span> so that we can account for numbers. Lunch is not included but there are plenty of <a href="https://www.estate.unsw.edu.au/food-retail/explore">coffee and food outlets</a> on campus.
+    </p>
+  <div id="a-lions" style="padding-top:20px">
+    <h4>Using Lions OS</h4>
```
Comment (Ivan-Velickovic) [inline Foundation/Summit/2024/index.html:194]:
```suggestion
    <h4>Using LionsOS</h4>
```

File: Foundation/Summit/2024/program.html [hunk 3]
```diff
@@ -356,16 +356,16 @@ <h4>Summit</h4>
   <tr>
     <th colspan="3">Day 4&emsp;18 October 2024&emsp;Training day @UNSW (TBC)&emsp;</th>
   </tr>
-  <tr>
-    <td></td>
+  <tr id="p-lions">
+    <td>10:00 - 12:30</td>
     <td>Training</td>
-    <td>Using Lions OS<br><span class="summit-abstract-presenter">Ivan Velickovic</span>,
+    <td><a href="index#a-lions">Using Lions OS</a><br><span class="summit-abstract-presenter">Ivan Velickovic</span>,
```
Comment (Ivan-Velickovic) [inline Foundation/Summit/2024/program.html:362]:
```suggestion
    <td><a href="index#a-lions">Using LionsOS</a><br><span class="summit-abstract-presenter">Ivan Velickovic</span>,
```

**Review by june-andronick — APPROVED** [review]

**Pull request merged**